tests/integration/validatorTests.py

```diff
@@ -39,55 +40,48 @@ def setUpClass(cls):
         column1.required = True
         column1.field_type = stringType
         column1.file_id = 1
-        cls.interfaces.validationDb.populateFile(column1)
```

Comment:
It looks like the validationManager does the filtering by file type, so by the time it gets passed to validate it only includes rules for the current file.  So the file type is probably not being used in these tests.